docs/os-intro.md

```diff
@@ -7,19 +7,33 @@ hide_title: true
 
 ![](/docs/assets/brand/aragonos.png)
 
-**aragonOS** is a **smart contract framework** for building DAOs, dapps and protocols.
+**aragonOS** is a **smart contract framework** for building decentralized organizations, dapps, and protocols.
 
-Its architecture is based on the idea of a decentralized organization being the aggregate of multiple components (called applications) and a pillar that is the **Kernel** which governs [how these applications can talk to each other](/docs/acl-intro.html) and how other entities can interact with them.
+--------
+
+As you may have guessed, **the OS in aragonOS stands for Operating System**. How so?
+
+A computer operating system manages how applications access the underlying resources of the computer, the hardware.
+
+aragonOS does the same for decentralized organizations or protocols. It abstracts away the management of how apps and external entities access the resources of the organization or protocol.
+
+These resources can be things such as assets, cryptocurrencies, the rights to claim a premium on a loan, or the rights to upgrade a smart contract.
+
+Its architecture is based on the idea of a decentralized organization or protocol being the aggregate of multiple components (called applications) connected by a pillar, called the **Kernel**, which are all governed by a special [Access Control List (ACL)](/docs/acl-intro.html) application that controls how these applications and other entities can interact with each other.
+
+--------
 
 aragonOS provides the following functionality:
 - **Upgreadability**: Smart contracts can be **upgraded to a newer version**. Example: *Fixing a bug or adding a feature*.
```

Comment:
I think we have too many styles in this section.  I think at least the italics should be removed.

Reply:
See 73dc3f3.